net.rpcs3.RPCS3.yaml

```diff
@@ -120,26 +72,34 @@ modules:
       ldflags: -fuse-ld=lld
     config-opts:
       - -DCMAKE_BUILD_TYPE=RelWithDebInfo
-      - -DBUILD_LLVM_SUBMODULE=OFF
+      - -DBUILD_LLVM_SUBMODULE=ON
       - -DUSE_NATIVE_INSTRUCTIONS=OFF
       - -DUSE_PRECOMPILED_HEADERS=OFF
       - -DUSE_SYSTEM_CURL=ON
       - -DUSE_SYSTEM_FFMPEG=ON
       - -DUSE_SYSTEM_LIBPNG=ON
       - -DUSE_SYSTEM_ZLIB=ON
+      - -DITTAPI_SOURCE_DIR=/run/build/rpcs3/ittapi
       - -Wno-dev
     post-install:
-      - |
-        set -eux;
+      - |-
+        set -eux
 
-        sed -i 's|<id>RPCS3</id>|<id>net.rpcs3.RPCS3</id>|' ${FLATPAK_DEST}/share/metainfo/rpcs3.metainfo.xml;
+        mkdir -p ${FLATPAK_DEST}/lib/ffmpeg # this path is checked at startup by rpcs3
```

Comment:
Was it necessary to move this line ?

Reply:
It was not necessary, but it can be done there instead of adding a global cleanup command.
Also more logical IMO, because it is not really about cleaning, but creating an empty path.

Reply:
I should perhaps create a separate commit for that.

Reply:
It's done like that in all other flatpaks for integrating ffmpeg-full. I would prefer to keep it the other way for consistency.

Reply:
Ok if done like that in other flatpaks, I revert this change.